universe/dataset.tt

```diff
@@ -0,0 +1,11 @@
+dataset @vikram.hello {
+    query (movie_query : String) := @vikram.hello.movie(query = movie_query)
+    #_[utterances=["$movie_query on tmdb",
+                    "tell me about the movie $movie_query",
+                    "search $movie_query",
+                    "what is $movie_query",
+                    "what happens in $movie_query",
+                    "${movie_query:const}"]]
```

Comment:
Utterances in `query` primitive templates should be noun phrases, so "tell", "search", "what is", are inappropriate. Given the noun phrases you specify, Genie automatically constructs all variations of questions and search commands.
In practice, you probably won't even need primitive templates because this is a plain database query, that should be done just with canonical forms.
(Look at Yelp for examples, or read the [Schema2QA paper](https://arxiv.org/abs/2001.05609) to learn more)